src/toast/ops/sim_ground.py

```diff
@@ -669,7 +705,7 @@ def _exec(self, data, detectors=None, **kwargs):
                 site=site,
             )
 
-            name = f"{scan.name}_{int(scan.start.timestamp())}"
+            name = f"{scan.name}-{scan.scan_indx}-{scan.subscan_indx}"
```

Comment:
As we discussed offline, this will invalidate the cached atmosphere sims for the ground benchmark.  I'll work on regenerating after this is merged.